tracer/Datadog.Trace.proj

```diff
@@ -80,7 +79,7 @@
 
   <!--  Used by CompileDependencyLibs-->
   <Target Name="BuildDependencyLibs">
-    <MSBuild Targets="Build" Projects="@(SampleLibProject);@(VbSampleLibProject);@(LiveDebuggerSampleLibProject)">
```

Comment:
Because it broke with the .NET 7 SDK 🤷‍♂️ Something to do with the path resolution for x64/x86/AnyCPU changed, which caused various things to break. The live debugger sampler is especially problematic as it needs to be referenced from both the AnyCPU projects (integration tests) and the x64/x86 samples.